example_case_folders/14_wind_standin_and_ldes/hercules_input_000.yaml

```diff
@@ -0,0 +1,66 @@
+# Name
+name: example_006
```

Comment:
Not a big deal, but it looks like `name` could be updated. We probably haven't been very consistent about that across the examples though.